### DISCUSSION TITLE
Watch Prompt in AI Pictures

### DISCUSSION THREAD

**Discussion by bisechen** [hbl917070/Tiefsee4#52] [Ideas]
Hi, AI-generated pictures have become more and more common in recent years. These AI-generated pictures should all contain (if they have not been deleted) prompt words and related setting parameters. Is it possible to add the function of viewing this kind of information in TieFsee?

I saw that XnView has similar functions, but it has too many functions, although it is still a very fast image viewing software.
![Uploading INxabpliT7.png…]()

**Comment by hbl917070** [comment]
It seems your image upload failed? I can’t view it.

Under normal circumstances, Tiefsee is able to display the prompt and related setting parameters used when generating AI images.

**Reply by bisechen** [reply]
It turns out that this option exists, but I didn’t look carefully to find it... I’m sorry, and thank you at the same time.

**Reply by tazztone** [reply]
hi i have a similar question.
there is no setting to show the metadata by default?
![image](https://github.com/user-attachments/assets/68c36c14-0de6-48d1-b1c9-2d3341bb68ce)
to avoid having to click this each time i change image.

**Reply by hbl917070** [reply]
Under normal circumstances, the parameters in ComfyUI are parsed into an easy-to-read format. The 'ComfyUI Data' field contains the raw data before parsing, which is difficult to read directly, so it is collapsed by default. However, ComfyUI has too many custom special nodes, and if parsing fails, it will revert to this unreadable raw data.

You can paste the original file of the image that failed to parse here, and I'll take a look to see what went wrong.

**Reply by tazztone** [reply]
i think the custom node easy ksampler is to blame that saved it like this.
here's the file
![xy-LoRA_00351_](https://github.com/user-attachments/assets/e674b47c-1e20-4fe2-9bc9-ebf7f431bbc9)
ye the metadata is a bit overkill here :D

**Reply by hbl917070** [reply]
I haven't used ComfyUI in a long time, so I'm not very familiar with the new nodes that require extra handling. For now, I've only done a simple fix for the easy fluxLoader node. 
The Nightly version should now be able to display the prompts for this image.
https://github.com/hbl917070/Tiefsee4/releases
